locations/spiders/dats24.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+import scrapy
+
+from locations.items import Feature
+
+
+class Dats24BESpider(scrapy.Spider):
+    name = "dats24_be"
+    item_attributes = {"brand": "DATS 24", "brand_wikidata": "Q15725576"}
+    start_urls = ["https://customer.dats24.be/wps/portal/datscustomer/fr/b2c/locator"]
+
+    def parse(self, response, **kwargs):
+        script_tags = response.text.split("<script")
+        for script_tag in script_tags:
+            if 'class="locatorMapData"' in script_tag:
+                start_index = script_tag.find(">") + 1
+                end_index = script_tag.find("</script>", start_index)
+                json_data = script_tag[start_index:end_index]
+                break
+
+        data = json.loads(json_data)
+        for store in data.get("stores"):
+            yield Feature(
+                {
+                    "ref": store.get("id"),
+                    "name": store.get("name"),
+                    "addr_full": " ".join(
+                        filter(
+                            None,
+                            [store.get("houseNumber"), store.get("street"), store.get("city"), store.get("postalCode")],
+                        )
+                    ),
```

Comment:
Can you put `houseNumber` and `street` together and set it to `street_address`, too? That field tends to be more useful than the `addr_full`.

Reply:
Thanks!